tools/reader_full_tsv_and_mpt_files.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 14 09:39:59 2022
+
+@author: Kim Degn Jensen
+Spectro Inlets A/S
+contact: support@spectroinlets.com
+"""
+
+# Call stuff
+from pathlib import Path
+import ixdat
+import os
+import glob
+
+#Function aimed at easy data readout
+def read_full_tsv_and_mpt_files(data_dir, tsv_function, tsv_reader_name, mpt_reader_name):
+
+    
+    # Define a nested function defning a parent directory for which you can and want to compiale multiple EC-MS experiements from. Note function is not selective in its input all .tsv and .mpt files iun the first subfolder layer is included 
+    def read_tsv_and_mpt_files_as_experiment_per_directories(rootdir):
```

Comment:
What does the word "experiment" mean here?
Is it equivalent to "measurement", as that word is used by ixdat and/or zilien?